plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/glue/TestHiveGlueMetastoreAccessOperations.java

```diff
@@ -87,6 +95,40 @@ public void cleanUpSchema()
         getQueryRunner().execute("DROP SCHEMA " + testSchema + " CASCADE");
     }
 
+    @Test
+    void testInsertOverwriteStatisticsDisabled()
```

Comment:
@raunaqmorarka I add the test to show case that the last commit won't influence the metastore operations